websites/I/iQIYi/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "SlowLife",
+      "id": "374905512661221377"
+    },
+    "url": [
+      "www.iq.com",
+      "tw.iqiyi.com"
+    ],
+    "description": {
+      "en": "Watch popular TV shows and movies online",
+      "zh_TW": "全球領先的線上影音視頻網站-正版高清影片影視免費線上看"
+    },
+    "service": "iQIYi",
+    "version": "1.0.0",
+    "logo": "https://imgur.com/a/RvkR8uN",
+    "thumbnail": "https://imgur.com/a/IBmFmza",
```

Comment:
you need the i.imgur links ending with .png/.jpg

Reply:
👍